packages/create-react-admin/templates/common/.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:react/jsx-runtime",
+        "plugin:react-hooks/recommended",
+        "prettier"
+    ],
+    "parser": "@typescript-eslint/parser",
+    "plugins": ["@typescript-eslint"],
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "settings": {
+        "react": {
+            "version": "detect"
+        }
+    },
+    "rules": {
+        "no-unused-vars": [
```

Comment:
why do we need custom rules?

Reply:
I think we'd better fix the code (or ignore the rule) in our authProvider instead

Reply:
Did you look at it? I'd rather not

Reply:
Disabled the rule locally